torch_frame/datasets/fake.py

```diff
@@ -87,6 +92,14 @@ def __init__(
                     arr[0::2] = len(arr[0::2]) * [np.nan]
                 df_dict[col_name] = arr
                 col_to_stype[col_name] = stype.text_embedded
+        if stype.text_tokenized in stypes:
+            for col_name in ['text_3', 'text_4']:
```

Comment:
would be good to rename this. text_tokenized_1, 2

also rename the previous text_1, 2 as text_embedded_1,2